Signal/test/textsecure/TSMessageStorageTests.m

```diff
@@ -57,16 +101,17 @@ - (void)testStoreIncomingMessage {
     TSIncomingMessage *newMessage = [[TSIncomingMessage alloc] initWithTimestamp:timestamp
```

Comment:
so again, I'd be a proponent for refactoring timestamps to indicate they are no longer timestamps but message ids at this stage. otherwise the code will get really unreadable as we also do have timestamps which are timestamps.